### PR TITLE
Revert "Fix MacOS 10.14 nothing rendering until resize"

### PR DESCRIPTION
**Pull request by LucioFranco**
This reverts commit 068100a.

### CODE REVIEW

**Comment by LucioFranco** [comment]
I've tested this reverted change on my macbook and it seems to work.

This is related to #1036.

**Comment by azriel91** [comment]
Yay software `s/with workarounds/that works/` 😃

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * `MaterialTextureSet` is removed as it is no longer needed. ([#1117])
 * `amethyst::core::Orientation` has been removed because of limited use. ([#1066])
 * `TimedDestroySystem` has been split into `DestroyAtTimeSystem` and `DestroyInTimeSystem`. ([#1129])
+* Reverted [MacOS OpenGL workaround][972] in favor of the upstream fix in `glutin`. ([#1184])
```
Comment (jojolepro) [inline docs/CHANGELOG.md]: 972 not a link

**Comment by Xaeroxe** [comment]
bors r= @azriel91 @jojolepro @Xaeroxe

**Pull request merged**